packages/zoe/src/contracts/oracle.js

```diff
@@ -0,0 +1,145 @@
+// @ts-check
+import { assert, details } from '@agoric/assert';
+import { E } from '@agoric/eventual-send';
+import { trade } from '../contractSupport';
+
+import '../../exported';
+
+/**
+ * This contract provides oracle queries for a fee or for pay.
+ *
+ * @type {ContractStartFn}
+ *
+ */
+const start = async zcf => {
+  const { oracleDescription } = zcf.getTerms();
```

Comment:
I think we should not include an `oracleDescription`. If it's used as an identity, it's subject to spoofing (I can create an oracle contract with a description "Official Chainlink Oracle" and people may believe it), and we should have people use petnames plus the contract instance as identification instead.